compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -47,6 +48,8 @@ int handleAutoAlgorithm(arser::Arser &arser, mpqsolver::bisection::BisectionSolv
 int entry(int argc, char **argv)
 {
   const std::string bisection_str = "--bisection";
+  const std::string patterns_str = "--patterns";
+  const std::string layernorm_str = "--LayerNorm";
```

Comment:
Could you update the option name?

```suggestion
  const std::string layernorm_str = "--q8_layernorm_with_q16_variance";
```

Reply:
> Could you update the option name?

Yes.